src/MountConfig.js

```diff
@@ -21,10 +21,33 @@ class MountConfig extends SchemaDerivedConfig {
         '^/mountpoints/.*$': 'mountpoint.schema.json',
       },
       handlers: {
-        '^/mountpoints$': NamedPairHandler('path', 'url'),
+        '^/mountpoints$': MountPointHandler(),
       },
     });
   }
+
+  /**
+   * Matches the given resource path against the mount points and returns the mount point if found.
+   * The mount point will also include a `relPath` property, which denotes the relative path
+   * from the mount points root.
+   *
+   * @param {string} resourcePath The resource path
+   * @returns {Mountpoint|null} The matched mount point or {@code null}.
+   */
+  match(resourcePath) {
+    const fullPath = resourcePath.endsWith('/') ? resourcePath : `${resourcePath}/`;
+
+    const mp = this.mountpoints.find((m) => fullPath.startsWith(m.path));
```

Comment:
please ensure that `mp.path` ends with `/`, otherwise it would also match for example `/msoft/docs`

Reply:
942160b should be the fix.